pkgs/top-level/all-packages.nix

```diff
@@ -19707,6 +19707,7 @@ with pkgs;
   catboost = callPackage ../development/libraries/catboost {
     # https://github.com/catboost/catboost/issues/2540
     cudaPackages = cudaPackages_11;
+    llvmPackagesCuda = llvmPackages_14;
```

Comment:
Yeah ideally we'd accommodate in `cudaPackages` (the same logic that handles gcc version ranges) but admittedly that might take a non-trivial refactoring and it's ok to postpone it. CC @NixOS/cuda-maintainers